ecfr-parser/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"sync"
+	"time"
+
+	"github.com/cmsgov/cmcs-eregulations/ecfr-parser/ecfr"
+	"github.com/cmsgov/cmcs-eregulations/ecfr-parser/parseXML"
+)
+
+const TIMELIMIT = 160 * time.Second
+
+func main() {
+	log.SetFlags(log.Lshortfile | log.LstdFlags)
+	title := 42
+
+	start := time.Now()
+
+	log.Println("[DEBUG] fetching Versions")
+	versions, err := ecfr.ExtractPartVersions(title)
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.Println(versions)
+
+	parts, err := ecfr.ExtractSubchapterParts(start, title, "IV", "C")
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.Println(parts)
+
+	var wg sync.WaitGroup
+	output := make(chan []byte)
+	go func() {
+		for obj := range output {
+			fmt.Println(string(obj))
+		}
+	}()
```

Comment:
Could be it's own function, this will be updated / removed when we get different kinds of output